openage/convert/hardcoded/texture.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+"""
+Constants for texture generation.
+"""
+
+MAX_TEXTURE_DIMENSION = 4096
+"""
+The maximum texture dimension that can be expected to be supported by all GPUs.
+"""
+
+MARGIN = 1
+"""
+A margin between subtextures in an atlas to avoid texture bleeding.
+"""
```

Comment:
hm, it's kinda ugly that they're written after what they document..
our docs are generated by doxygen, so it would be nice if stuff was [parsable by that](https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#pythonblocks), so everything should be python standard, but in this case it'd be `## blabla doc` before the `MARGIN = 1`.